server/scsynth/SC_Jack.cpp

```diff
@@ -446,9 +453,9 @@ void SC_JackDriver::Run() {
 
         // main loop
 #ifdef SC_JACK_USE_DLL
-        int64 oscTime = mOSCbuftime = (int64)((mDLL.PeriodTime() + mMaxOutputLatency) * kSecondsToOSCunits + .5);
+        int64 oscTime = mOSCbuftime = (uint64)((mDLL.PeriodTime() + mMaxOutputLatency) * kSecondsToOSCunits + .5);
         // 		int64 oscInc = mOSCincrement = (int64)(mOSCincrementNumerator / mDLL.SampleRate());
-        int64 oscInc = mOSCincrement = (int64)((mDLL.Period() / numBufs) * kSecondsToOSCunits + .5);
+        int64 oscInc = mOSCincrement = (uint64)((mDLL.Period() / numBufs) * kSecondsToOSCunits + .5);
```

Comment:
why did you change these casts? both of the things you're assigning to are `int64`

Reply:
all done

Reply:
still not sure about this one, can you explain the change?

Reply:
This code is no more active!!

Reply:
sorry, i really don't understand your response. why change the cast from `(int64)` to `(uint64)`?

Reply:
sorry, I thought this code was not present!!
corrected now.